openmc/geometry.py

```diff
@@ -455,3 +440,16 @@ def get_lattices_by_name(self, name, case_sensitive=False, matching=False):
         lattices = list(lattices)
         lattices.sort(key=lambda x: x.id)
         return lattices
+
+    def determine_paths(self):
+        """Count the number of instances for each cell in the Geometry, and
```

Comment:
Should this also note that the materials are counted and recorded in the `Material.num_instances` property?